conf/modules/digital_cam_pprzlink.xml

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE module SYSTEM "./module.dtd">
+
+<module name="digital_cam_pprzlink" dir="digital_cam">
+  <doc>
+    <description>
+      Digital Photo Camera Triggering using PPRZLINK messages
```

Comment:
It appears to be able to shoot and set the exposure. Could you add some info of which camera/software/... is at the receiving end of this interface?

Reply:
done